### ISSUE TITLE
Remove Forking of StyleSheetValidation.js When We Reach 6/25 Facebook Nightly Build

### ISSUE THREAD

**Issue by NickGerleman** [microsoft/react-native-windows#5269]
The button component defines a style as so:

```jsx
const styles = StyleSheet.create({
  button: Platform.select({
    ios: {},
    android: {
      elevation: 4,
      // Material design blue from https://material.google.com/style/color.html#color-color-palette
      backgroundColor: '#2196F3',
      borderRadius: 2,
    },
  }),
```

For non ios/Android platforms, that creates a style object which looks like:
```js
{
  button: undefined,
  ...
}
```

As of 0.63(ish) `StyleSheet.create` will choke on undefined values. This is caused by this commit https://github.com/facebook/react-native/commit/f464dad5d4f0fbf1cb23e21d22907ffddeaf97e4

We should either change that commit or change Button to have a better fallback.

**Comment by NickGerleman** [comment]
Sounds like the style validation changes were unintentional. Going to create a PR to fix those and patch that in instead.

**Comment by NickGerleman** [comment]
PR to fix the style validation issue: https://github.com/facebook/react-native/pull/29171

**Comment by chrisglein** [comment]
If we take that other PR we don't need to do this for 0.63. But will need to one of them so putting this in 63 until we iron that out.

**Comment by NickGerleman** [comment]
Going to go with the style validation change instead. Going to repurpose this bug for that.

**Comment by chrisglein** [comment]
@NickGerleman Sounds like we took the PR? Does the StyleSheetValidation fork need to be addressed for 0.63 or is it okay to push out?

**Comment by NickGerleman** [comment]
The eventual solution was to fork StyleSheetValidation and submit the fix upstream. It was accepted into the 6/25 build, so we will be able to remove this in the 0.64 timeframe. Will update the milestone.
